src/components/Projects/WBS/WBSDetail/WBSTasks.jsx

```diff
@@ -29,21 +29,51 @@ const WBSTasks = props => {
   const [isShowImport, setIsShowImport] = useState(false);
   const [selectedId, setSelectedId] = useState(null);
   const [filterState, setFilterState] = useState('all');
+  const [openAll, setOpenAll] = useState(false);
+
+  const load = async()=>{
+    const le = [0,1,2,3,4];
```

Comment:
When possible please rename the variables as to improve their readability and funcionality. Thank you.